app/views/jst/developer_key.handlebars

```diff
@@ -42,9 +42,19 @@
     </td>
     <td>
         <div>
+          {{#t "access_token_count"}}Access Token Count: {{access_token_count}}{{/t}}
```

Comment:
Yeah, I was just going with the style that was already being used in the file.  I can see the new way documented in https://github.com/instructure/canvas-lms/wiki/I18n#handlebars.

I'll change my additions to remove the explicit keys. Do you want me to change the existing ones while I'm at it?